from cloudify import ctx

ctx.retry_operation(message='Retry')
